hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/TestSources.java

```diff
@@ -161,4 +187,59 @@ void fillBuffer(TimestampedSourceBuffer<T> buf) throws Exception {
             }
         }
     }
+
+    /**
+     * Returns a {@link com.hazelcast.jet.pipeline.StreamSource} for {@code
```

Comment:
javadoc lacks detail about what timestamps the items will have

Reply:
Information got extended with d476f97f6c1cb3bdd945b2b0da124869086022da